openreview/journal/invitation.py

```diff
@@ -3222,487 +3626,764 @@ def set_comment_invitation(self, note):
+            signatures=[self.journal.venue_id]
+        )
 
-    def set_authors_deanonymization_invitation(self, note):
+    def set_authors_deanonymization_invitation(self):
         venue_id = self.journal.venue_id
         editors_in_chief_id = self.journal.get_editors_in_chief_id()
-        paper_authors_id = self.journal.get_authors_id(number=note.number)
 
-        authors_deanonymization_invitation_id = self.journal.get_authors_deanonymization_id(number=note.number)
+        invitation_content = {
+            'process_script': {
+                'value': self.get_process_content('process/authors_deanonimization_process.py')
+            }                
+        }
+        edit_content = {
+            'noteNumber': { 
+                'value': {
+                    'param': {
+                        'regex': '.*', 'type': 'integer' 
+                    }
+                }
+            },
+            'noteId': { 
+                'value': {
+                    'param': {
+                        'regex': '.*', 'type': 'string' 
+                    }
+                }
+            },
+            'duedate': { 
+                'value': {
+                    'param': {
+                        'regex': '.*', 'type': 'integer' 
+                    }
+                }
+            }
+        }
 
-        invitation = Invitation(id=authors_deanonymization_invitation_id,
-            invitees=[venue_id, paper_authors_id],
-            readers=['everyone'],
-            writers=[venue_id],
-            signatures=[venue_id],
-            maxReplies=1,
-            edit={
-                'signatures': { 'const': [ paper_authors_id ] },
-                'readers': { 'const': [ venue_id, paper_authors_id ] },
-                'writers': { 'const': [ venue_id ] },
+        invitation = {
+            'id': self.journal.get_authors_deanonymization_id(number='${2/content/noteNumber/value}'),
+            'invitees': [venue_id, self.journal.get_authors_id(number='${3/content/noteNumber/value}')],
+            'readers': ['everyone'],
+            'writers': [venue_id],
+            'signatures': [venue_id],
+            'maxReplies': 1,
+            'edit': {
+                'signatures': [ self.journal.get_authors_id(number='${4/content/noteNumber/value}') ],
+                'readers': [ venue_id, self.journal.get_authors_id(number='${4/content/noteNumber/value}') ],
+                'writers': [ venue_id ],
                 'note': {
-                    'signatures': { 'const': [ paper_authors_id ] },
-                    'forum': { 'const': note.id },
-                    'replyto': { 'const': note.id },
-                    'readers': { 'const': [ editors_in_chief_id, paper_authors_id ] },
-                    'writers': { 'const': [ venue_id ] },
+                    'signatures': [ self.journal.get_authors_id(number='${5/content/noteNumber/value}') ],
+                    'forum':  note.id,
```

Comment:
This note it not defined anywhere since the function does not take it as a parameter anymore

Reply:
Fixed!